### PR TITLE
Add document for tiup cluster rename

### PR DESCRIPTION
**Pull request by lucklove**
Signed-off-by: lucklove <gnu.crazier@gmail.com>

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: maintain-tidb-using-tiup.md [hunk 1]
```diff
@@ -169,6 +169,21 @@ tiup cluster patch test-cluster /tmp/tidb-hotfix.tar.gz -R tidb
...
 ```
 
+## 重命名集群
+
+在集群部署并且启动之后，可以通过 `tiup cluster rename` 命令来对集群重命名：
+
+{{< copyable "shell-regular" >}}
+
+```bash
+tiup cluster stop ${cluster-name} ${new-name}
```
Comment (lonng) [inline maintain-tidb-using-tiup.md:179]: 请先 self-review。

File: maintain-tidb-using-tiup.md [hunk 2]
```diff
@@ -169,6 +169,21 @@ tiup cluster patch test-cluster /tmp/tidb-hotfix.tar.gz -R tidb
 tiup cluster patch test-cluster /tmp/tidb-hotfix.tar.gz -N 172.16.4.5:4000
 ```
 
+## 重命名集群
+
+在集群部署并且启动之后，可以通过 `tiup cluster rename` 命令来对集群重命名：
```
Comment (TomShawn) [inline maintain-tidb-using-tiup.md:174]:
```suggestion
部署并启动集群后，可以通过 `tiup cluster rename` 命令来对集群重命名：
```

File: maintain-tidb-using-tiup.md [hunk 3]
```diff
@@ -169,6 +169,21 @@ tiup cluster patch test-cluster /tmp/tidb-hotfix.tar.gz -R tidb
...
+{{< copyable "shell-regular" >}}
+
+```bash
+tiup cluster rename ${cluster-name} ${new-name}
+```
+
+> **注意：**
+> 
+> 1. 重命名集群会重启监控（Prometheus 和 Grafana）
+> 2. 重命名集群之后 Grafana 会残留个别老集群名字的 panel，需要手动删除
```
Comment (TomShawn) [inline maintain-tidb-using-tiup.md:185]:
```suggestion
> + 重命名集群之后 Grafana 可能会残留一些旧集群名的面板，需要手动删除这些面板。
```

File: maintain-tidb-using-tiup.md [hunk 4]
```diff
@@ -169,6 +169,21 @@ tiup cluster patch test-cluster /tmp/tidb-hotfix.tar.gz -R tidb
...
+
+{{< copyable "shell-regular" >}}
+
+```bash
+tiup cluster rename ${cluster-name} ${new-name}
+```
+
+> **注意：**
+> 
+> 1. 重命名集群会重启监控（Prometheus 和 Grafana）
```
Comment (TomShawn) [inline maintain-tidb-using-tiup.md:184]:
```suggestion
> + 重命名集群会重启监控（Prometheus 和 Grafana）。
```

**Pull request merged**